sssom/util.py

```diff
@@ -266,12 +269,15 @@ def assign_default_confidence(
     :return: A Tuple consisting of the original DataFrame and dataframe consisting of empty confidence values.
     """
     # Get rows having numpy.NaN as confidence
-    if df is not None and "confidence" not in df.columns:
-        df["confidence"] = np.NaN
-
-    nan_df = df[df["confidence"].isna()]
-    if nan_df is None:
-        nan_df = pd.DataFrame(columns=df.columns)
+    if df is not None:
+        if CONFIDENCE not in df.columns:
+            df[CONFIDENCE] = np.NaN
```

Comment:
Will this lead to merged dataframes having confidence columns even if neither of the two merged data frames had one to begin with?